cli/cmd/lql.go

```diff
@@ -19,71 +19,217 @@
 package cmd
 
 import (
+	"fmt"
+	"io"
```

Comment:
io package won't be used when line 206 is changed

```suggestion
```

Reply:
Fixed.